dtests/src/test/java/io/snappydata/hydra/cluster/SnappyLocatorHATest.java

```diff
@@ -94,10 +105,20 @@ protected static void ddlOpAfterLocatorStop_ClusterRestart(String vmDir, String
       conn = getServerConnection();
       conn.createStatement().executeUpdate(query);
       Log.getLogWriter().info("query executed successfully: " + query);
-      /*rs = conn.createStatement().executeQuery("select * from tab1 where 1=0");
-      StructTypeImpl sti = ResultSetHelper.getStructType(rs);
-      List<Struct> queryResult = ResultSetHelper.asList(rs, sti, false);
-      Log.getLogWriter().info("Result for query : " + query + "\n" + queryResult.toString());*/
+      query = "insert into tab1 values(111, 'aaa', 'hello')";
```

Comment:
We can define the insert statements in a string, and use them. I see them defined at multiple places.